detect/tasks/sma.go

```diff
@@ -60,46 +63,27 @@ func (h *SMAHandler) httpAvailable(ip string) bool {
 func (h *SMAHandler) Test(log *util.Logger, in ResultDetails) (res []ResultDetails) {
 	h.mux.Lock()
 
-	if h.listener != nil {
+	if h.handled {
 		h.mux.Unlock()
 		return nil
 	}
 
-	var err error
-	if h.listener, err = sma.New(log, ""); err != nil {
+	devices, err := sunny.DiscoverDevices(h.Password)
+	if err != nil {
 		log.ERROR.Println("shm:", err)
```

Comment:
s/shm/sma?

Reply:
shm hieß das bisher kann ich aber auch in sma ändern

Reply:
ich hab das jetzt alles von shm auf sma geänder (auch in den structs)